runtime.md

```diff
@@ -15,3 +15,51 @@ Runs a process in a container. Can be invoked several times.
 Not sure we need that from oc cli. Process is killed from the outside.
 
 This event needs to be captured by oc to run onstop event handlers.
+
+## Hooks
+Hooks allow one to run code before/after various lifecycle events of the container. The state of the container is passed to the hooks over stdin, so the hooks could get the information they need to do their work.
+
+Hooks are executed from the host's filesystem. Hook paths can be:
+* An absolute path,
+* a file name (without path separators), in which case the file is searched for within the PATH environment variable, or
+* a relative path, in which case the file is found relative to the directory containing the config file. Relative paths cannot reference parent directories (i.e., no ../ ...).
```

Comment:
You probably want backticks, and may want an ellipsis character after the slash: `../…`.  If you don't want non-ASCII characters, I'd probably just say “no `../`”.